src/resources/events-resource.js

```diff
@@ -32,6 +32,29 @@ export default function EventsResource({apiHandler}) {
             return apiHandler.getAll(`events/${eventType}/rules/history`, params);
         },
 
+        getAllTimelineMessages({eventType = null, limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
+            const params = {
+                limit,
+                offset,
+                sort,
+                filter,
+                q
+            };
+            return apiHandler.getAll(`events/${eventType}/timeline`, params);
+        },
+
+        getTimelineMessage({eventType, messageId}) {
+            return apiHandler.get(`events/${eventType}/timeline/${messageId}`);
+        },
+
+        deleteTimelineMessage({eventType, messageId}) {
+            return apiHandler.delete(`events/${eventType}/timeline/${messageId}`);
+        },
+
+        createTimelineComment({eventType, data}) {
+            return apiHandler.create(`events/${eventType}/timeline`, '', data);
```

Comment:
I wonder why not `apiHandler.post`?

Reply:
All the other resource that have a timeline use `createTimelineComment` with `apiHandler.create` - I decided to keep up with the convention.

Reply:
The code generator would use `post` here.
Explanation: 
In other resources we have the special method `create` that internally uses `put/post` depending if the `id` is provided as a parameter or not. 

In this concrete case, we don't have `id` parameter because there is no `put` in `users api-definitions` that's why the generator will generate this code: 

```
 createTimelineComment({eventType, data}) {
      return apiHandler.post(`events/${eventType}/timeline`, data);
    },
```